system_tests/bigtable.py

```diff
@@ -93,3 +93,59 @@ def test_list_clusters(self):
             cluster_existence = (cluster in EXISTING_CLUSTERS or
                                  cluster == Config.CLUSTER)
             self.assertTrue(cluster_existence)
+
+    def test_reload(self):
+        # Use same arguments as Config.CLUSTER (created in `setUpModule`).
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, CLUSTER_ID)
+        # Make sure metadata unset before reloading.
+        cluster.display_name = None
+        cluster.serve_nodes = None
+
+        cluster.reload()
+        self.assertEqual(cluster.display_name, Config.CLUSTER.display_name)
+        self.assertEqual(cluster.serve_nodes, Config.CLUSTER.serve_nodes)
+
+    def test_create_cluster(self):
+        cluster_id = '%s-a' % (CLUSTER_ID,)
+        cluster = Config.CLIENT.cluster(CENTRAL_1C_ZONE, cluster_id)
+        operation = cluster.create()
+        # Make sure this cluster gets deleted after the test case.
+        self.clusters_to_delete.append(cluster)
+
+        # We want to make sure the operation completes.
```

Comment:
It usually takes about 0.5 seconds (based on `request_time` and `finish_time` in the `CreateClusterMetadata`), I have no idea why the API doesn't just wait until completion (instead of returning a long-running operation).

As for waiting for the operation to complete, wouldn't that create an infinite loop?

I was "happy enough" with the 0.5 second empirically observed completion time to make 2 seconds a threshold for the test.

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/7c77c170c8818a0440e0d2f4ecedcb8329c1f8d9/bigtable-protos/src/main/proto/google/bigtable/admin/cluster/v1/bigtable_cluster_service_messages.proto#L88-L91